hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/tenantconfiguration/TenantConfigurationDashboardView.java

```diff
@@ -265,93 +129,27 @@ private HorizontalLayout saveConfigurationButtonsLayout() {
-        }
-
-        if (configWindowBean.isActionAutocleanup()) {
-            writeConfigOption(ACTION_CLEANUP_ACTION_STATUS, configWindowBean.getActionCleanupStatus().getStatus()
-                    .stream().map(Action.Status::name).collect(Collectors.joining(",")));
-
-            writeConfigOption(TenantConfigurationKey.ACTION_CLEANUP_ACTION_EXPIRY,
-                    TimeUnit.DAYS.toMillis(Long.parseLong(configWindowBean.getActionExpiryDays())));
-        }
-        if (!readConfigOption(TenantConfigurationKey.MULTI_ASSIGNMENTS_ENABLED)) {
-            writeConfigOption(TenantConfigurationKey.REPOSITORY_ACTIONS_AUTOCLOSE_ENABLED,
-                    configWindowBean.isActionAutoclose());
-        }
-
-        if (configWindowBean.isMultiAssignments()
-                && !readConfigOption(TenantConfigurationKey.MULTI_ASSIGNMENTS_ENABLED)) {
-            writeConfigOption(TenantConfigurationKey.MULTI_ASSIGNMENTS_ENABLED, configWindowBean.isMultiAssignments());
-            repositoryConfigurationView.disableMultipleAssignmentOption();
-        }
-        writeConfigOption(TenantConfigurationKey.AUTHENTICATION_MODE_HEADER_ENABLED,
-                configWindowBean.isCertificateAuth());
-        if (configWindowBean.isCertificateAuth()) {
-            final String value = configWindowBean.getCaRootAuthority() != null ? configWindowBean.getCaRootAuthority()
-                    : "";
-            writeConfigOption(TenantConfigurationKey.AUTHENTICATION_MODE_HEADER_AUTHORITY_NAME, value);
-        }
-
-        if (configWindowBean.isPollingTime()) {
-            tenantConfigurationManagement.addOrUpdateConfiguration(TenantConfigurationKey.POLLING_TIME_INTERVAL,
-                    DurationHelper.durationToFormattedString(configWindowBean.getPollingTimeDuration()));
-        } else {
-            tenantConfigurationManagement.deleteConfiguration(TenantConfigurationKey.POLLING_TIME_INTERVAL);
-        }
-
-        if (configWindowBean.isPollingOverdue()) {
-            tenantConfigurationManagement.addOrUpdateConfiguration(TenantConfigurationKey.POLLING_OVERDUE_TIME_INTERVAL,
-                    DurationHelper.durationToFormattedString(configWindowBean.getPollingOverdueDuration()));
-        } else {
-            tenantConfigurationManagement.deleteConfiguration(TenantConfigurationKey.POLLING_OVERDUE_TIME_INTERVAL);
-        }
-
-        populateAndGetSystemConfig();
-    }
-
-    private boolean readConfigOption(final String configurationKey) {
-        final TenantConfigurationValue<Boolean> enabled = tenantConfigurationManagement
-                .getConfigurationValue(configurationKey, Boolean.class);
-
-        return enabled.getValue() && !enabled.isGlobal();
-    }
-
-    private <T extends Serializable> void writeConfigOption(final String key, final T value) {
-        tenantConfigurationManagement.addOrUpdateConfiguration(key, value);
+        autowiredConfigurationViews.forEach(ConfigurationGroup::save);
```

Comment:
maybe replace autowiredConfigurationViews with the filtered List. Doublecheck first

Reply:
Done. Removed whole method `saveSystemConfigBean` due to redundancy.
The filtered list is used and was also used before.